app/src/main/java/com/pajato/android/gamechat/exp/BaseExperienceFragment.java

```diff
@@ -76,14 +86,54 @@ public boolean getTurn() {
         return mTurn;
     }
 
-    /** Remove this after dealing with the chess and checkers fragments. */
+    /** Remove this after dealing with the chess_exp and checkers_exp fragments. */
     abstract public void messageHandler(final String message);
 
+    /** Create a new experience to be displayed in this fragment. */
+//    abstract protected void createExperience(@NonNull final Context context,
+//                                    @NonNull final Dispatcher<ExpFragmentType, ExpProfile> dispatcher);
+
     // Protected instance methods.
 
-    /** Create a new experience to be displayed in this fragment. */
-    protected void createExperience(final Context context, final Dispatcher<ExpFragmentType, ExpProfile> dispatcher) {
-        // nop; the subclass should handle this.
+    /** Return either a null placeholder key value or a sentinel value as the experience key. */
+    protected String getExperienceKey() {
+        // Determine if there is a signed in account.  If so use the null placeholder.
+        String accountId = AccountManager.instance.getCurrentAccountId();
```

Comment:
Recently I've taken to using AccountManager.instance.hasAccount() more often than not.